beta-src/src/utils/map/WDCountryHighlightFilters.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from "react";
+import webDiplomacyTheme from "../../webDiplomacyTheme";
+
+interface CountryHighlightFilter {
+  countryName: string;
```

Comment:
We should make this a country enum, and then reverse map the country enum when setting the id.